src/System/Router/Route.tsx

```diff
@@ -1,35 +1,38 @@
 import { RouteSpinner } from "System/Relay/renderWithLoadProgress"
 import { RouteConfig, HttpError, Match } from "found"
 import BaseRoute from "found/Route"
-import * as React from "react";
-import { CacheConfig, GraphQLTaggedNode } from "relay-runtime";
-import { ArtsyRequest, ArtsyResponse } from "Server/middleware/artsyExpress";
-import { NextFunction } from "express";
+import * as React from "react"
+import { CacheConfig, GraphQLTaggedNode } from "relay-runtime"
+import { ArtsyRequest, ArtsyResponse } from "Server/middleware/artsyExpress"
+import { NextFunction } from "express"
+import { LayoutVariant } from "Apps/Components/Layouts"
+import { RemoveIndex } from "Utils/typeSupport"
 
 interface RouteConfigProps extends RouteConfig {
   cacheConfig?: CacheConfig
   children?: AppRouteConfig[]
-  displayFullPage?: boolean
   fetchIndicator?: FetchIndicator
-  hideNav?: boolean
-  hideFooter?: boolean
+  /** FIXME: Remove. Avoid poluting global route config with application specific concerns */
   hideNavigationTabs?: boolean
   ignoreScrollBehavior?: boolean
   ignoreScrollBehaviorBetweenChildren?: boolean
+  layout?: LayoutVariant
```

Comment:
Should this be required?

Reply:
No, just defaults to the default layout.

Reply:
ehh scratch that, fallback is better